enhancements/observability/multicluster-observability-addon-interoperability.md

```diff
@@ -0,0 +1,217 @@
+---
+title: multicluster-logs-traces-forwarding
+authors:
+  - "@pavolloffay"
+reviewers:
+  - "@moadz"
+  - "@periklis"
+  - "@alanconway"
+  - "@jcantrill"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@moadz"
+  - "@periklis"
+  - "@alanconway"
+  - "@jcantrill"
+api-approvers:
+  - "@moadz"
+  - "@periklis"
+  - "@alanconway"
+  - "@jcantrill"
+creation-date: 2024-06-08
+last-updated: 2024-06-08
+tracking-link:
+  -
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+
+# Multi-Cluster telemetry data exportation
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The objective of multi cluster observability is to offer users a capability to collect 
+metrics, logs and traces from spoke clusters. Currently, the collection technology 
+uses three different technology stacks and protocols for exporting data (Prometheus 
+remote-write for metrics, Loki push for logs and OTLP for traces). 
+Loki push and Prometheus remote-write are not commonly supported as ingest protocols by 
+observability vendors.
+
+This enhancement proposal seeks to strengthen interoperability of MCOA by unifying and 
+simplifying exporting of all MCOA telemetry data (metrics, logs, traces) 
+by exposing a unified export API and consolidating export protocols. This capability
+enables users to send data from MCOA to any observability vendor and apply 
+fine-grained filtering and routing on exported data to configurable sinks.
+
+## Motivation
+
+At the moment exporting all telemetry data from OpenShift is fragmented to tree 
+technology stacks (Prometheus, ClusterLoggingForwarder, OpenTelemetry collector). 
+Every tool uses a different configuration API, export protocol and provides (or does 
```

Comment:
@saswatamcode This may be true but I don't read that as the intent of this proposal; that seems orthogonal.  I understand the intent as providing a way to unify the signals to a single protocol which is OTLP using OTEL semantics